client/controllers/events/eventIncidentReports.coffee

```diff
@@ -53,7 +53,7 @@ Template.eventIncidentReports.onCreated ->
     # indicator to appear indicating filters have been considered in results.
     data = @data
     data.filterQuery.get()
-    if data.loaded.get()
+    if data.loaded and data.loaded.get()
```

Comment:
For future reference should we use this pattern or the existential operator? I suppose this compiles down to less code.